README.md

```diff
@@ -35,6 +41,17 @@ Set your Github token to query API without strict rate limiting,
 export GITHUB_API_TOKEN=<your-github-token>
 ```
 
+Set remote RPC URL to validate contract bytecode at remote rpc node,
+
+```bash
+export REMOTE_RPC_URL =<remote-rpc-url>
+```
+
+Set local RPC URL to validate contract bytecode at local rpc node (Hardhat),
```

Comment:
This description is misleading, we need to have LOCAL_RPC_URL mostly to address the case when immutables should be checked against the local deployment and provided ctor arguments, isn't it?

Reply:
Set local RPC URL to check immutables against the local deployment and provided constructor arguments